browser-test/src/support/applicant_questions.ts

```diff
@@ -72,6 +72,19 @@ export class ApplicantQuestions {
     await this.page.click(`#continue-application-button`);
   }
 
+  async expectProgramExist(programName: string, description: string) {

```

Comment:
Nit: maybe "expectProgramVisible" and below "expectProgramNotVisible" or something instead?

Reply:
Done